documentation/book/cluster-operator.adoc

```diff
@@ -515,6 +518,27 @@ Furthermore, containers doing lots of disk I/O (such as Kafka broker containers)
 for use as operating system page cache. On such containers, the request memory should be substantially more than the
 memory used by the JVM.
 
+[[kafka_rack]]
+===== Kafka rack
```

Comment:
You have to update the section about RBAC rights. Explain why we need what we need. This was needed by a user, we should keep it updated! You need also to explain that users need to change the namespace. Perhaps we should even use some script now for installing this, because the examples now do not work everywhere and do not fulfil the installation purpose anymore.